subsys/net/lib/openthread/platform/radio.c

```diff
@@ -98,6 +102,25 @@ static otError tx_result;
 K_FIFO_DEFINE(rx_pkt_fifo);
 K_FIFO_DEFINE(tx_pkt_fifo);
 
+static int8_t get_transmit_power_for_channel(uint8_t aChannel)
+{
+	int8_t channelMaxPower = OT_RADIO_POWER_INVALID;
```

Comment:
Please use `snake_case` for Zephyr code.

Reply:
fxed.